sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/entity/Student.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * 
+ */
+package com.consoleconnect.mw.sonarqube.sample.entity;
+
+/**
+ * @author dxiong
+ *
+ */
+public class Student {
+
+  
+  public String name() {
+    return"name";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sq.ci.consoleconnect.com/coding_rules#rule_key=squid%3AS3400)